tools/dbus_ignorelist

```diff
@@ -877,6 +877,7 @@ org.freedesktop.systemd1.Mount.LogsDirectoryMode
 org.freedesktop.systemd1.Mount.LogsDirectorySymlink
 org.freedesktop.systemd1.Mount.ManagedOOMMemoryPressure
 org.freedesktop.systemd1.Mount.ManagedOOMMemoryPressureLimit
+org.freedesktop.systemd1.Mount.ManagedOOMMemoryPressureDurationUSec
```

Comment:
No, this is for properties that require no version info (i.e. old enough to be assumed available everywhere)